modules/node_modules/@ncigdc/dux/versionInfo.js

```diff
@@ -1,48 +1,33 @@
 // @flow
-/* global __GDC_API__, __COMMIT_HASH__, __VERSION_TAG__, __COMMIT_LINK__ */
-
 import { fetchApi } from '@ncigdc/utils/ajax';
 import { handleActions } from 'redux-actions';
-import packageJSON from './package.json';
+import packageJSON from '../../../../package.json';
```

Comment:
Why wouldn't it be bundled if it's imported? 
Isn't deploying just using the build script in the package.json?